example/.gitignore

```diff
@@ -0,0 +1,7 @@
+.travis.yml
+README.md
+coq-chapar-stores.opam
+coq-chapar.opam
+default.nix
+index.html
+index.md
```

Comment:
Maybe we should neither ignore `index.md` or remove it in `make clean`, as this file is subject to changes that repos should keep track of.